packages/vue-dot/src/elements/DialogBox/DialogBox.vue

```diff
@@ -38,14 +38,14 @@
 						v-bind="options.cancelBtn"
 						@click="$emit('cancel')"
 					>
-						{{ locales.cancel }}
+						{{ buttonLabels.cancel?buttonLabels.cancel:locales.cancel }}
 					</VBtn>
 
 					<VBtn
 						v-bind="options.confirmBtn"
 						@click="$emit('confirm')"
 					>
-						{{ locales.confirm }}
+						{{ buttonLabels.confirm?buttonLabels.confirm:locales.confirm }}
```

Comment:
Je pense qu'il faudrait utiliser 2 props distinctes pour simplifier ici
```suggestion
						{{ confirmBtnLabel }}
```